pages/index.js

```diff
@@ -27,13 +27,15 @@ function Header({ show }) {
 function Body({ txId, data, status }) {
   if (status === 200) {
     const maybeSigs = useAsync(fetchSignatures, [txId]);
-    const [clientSigList, setClientSigList] = React.useState([])
+    const [clientSigList, setClientSigList] = React.useState([]);
+    const [signed, setSigned] = React.useState(false);
```

Comment:
What's the role of this new variable? How does it manage the visibility different than the way visibility was previously managed?

Reply:
I got rid of it, is was originally just a way to "force update" but I found a better way.